.env.samle

```diff
@@ -0,0 +1,3 @@
+DB_HOST='environment'
```

Comment:
Could you please:
1. insert missed `p` in the file name `.env.samle`
2. use corresponding variable names, like
```
API_KEY='your api key'
DELIVERY_TOKEN='your delivery token'
DEVELOPMENT_ENVIRONMENT='your environment`
```